### DISCUSSION TITLE
Accessibility

### DISCUSSION THREAD

**Discussion by ScottV12** [community/community#142194] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I need to learn more about coding, some can help me, send me message to my inbox I do pay for the one who can teach me!

**Comment by harshakalluri1403** [comment]
Yeah please ping me my git prof name is harshakalluri14

**Comment by Dipeshdahiya** [comment]
Hi @ScottV12 

If you're looking to learn more about coding, GitHub offers some great free resources to get started, such as GitHub Learning Lab, where you can follow tutorials and projects at your own pace.

You can also check out platforms like Codecademy, freeCodeCamp, and Coursera, which provide structured courses in various programming languages and concepts.

If you're interested in personalized one-on-one tutoring, you may want to consider platforms like Upwork or Fiverr, where you can hire experienced developers for lessons. Alternatively, sharing more details about what you want to learn (languages, frameworks, etc.) could help others in the community point you in the right direction.

Good luck on your coding journey!

**Comment by ebndev** [comment]
Hi @ScottV12, we’re glad you’re here 😀

Unfortunately, we currently do not allow solicitation or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

Feel free to check out the pinned posts in our [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Programming+Help%22) category to get you started on your coding journey!
